templates/commands/render_action_include_test.go

```diff
@@ -76,14 +76,14 @@ func TestActionInclude(t *testing.T) {
 				Paths: modelStrings([]string{"{{.my_dir}}/{{.my_file}}"}),
 			},
 			templateContents: map[string]modeAndContents{
-				"foo/bar.txt": {0o600, "file contents"},
+				filepath.Join("foo", "bar.txt"): {0o600, "file contents"},
```

Comment:
As we just discussed offline, it would be much easier on the eyes in my opinion to leave these as forward slashes and translate them to filepath.Separator below.